app/components/dsfr_component/tile_component.rb

```diff
@@ -6,21 +6,24 @@ class TileComponent < DsfrComponent::Base
     # @param image_alt [String] L'alternative de l'image. Doit a priori être vide si l'image est illustrative et n'apporte pas de valeur sémantique.
     # @param description [String] description (optional)
     # @param orientation [String] :horizontal or :vertical
-    def initialize(title:, url:, image_src: nil, image_alt: "", description: nil, orientation: :vertical, classes: [], html_attributes: {})
+    # @param title_tag [Symbol] :h1, :h2, :h3, :h4 (default), :h5, :h6
```

Comment:
I prefer your argument of using `heading_level: number`. It's the easiest we can make the interface, and it keeps more semantic on our side which I think is nice.

Reply:
fine with me, I don’t have a strong opinion on this, I'll update it